### PR TITLE
Feature/metobs class names

### PR DESCRIPTION
**Pull request by mgcth**
**Description**

Class names in metobs are long and complex. If we want to expose them to users, we should simplify them.

**Proposal**

Simplify class names in Metobs module. Strip `Metobs` and `V1` from the name. If version 2 ever comes out, we can come back to this.

### CODE REVIEW

File: tests/unit/test_unit_metobs.py [hunk 1]
```diff
@@ -422,7 +422,7 @@ def test_unit_metobsparameterv1_init(
 
 
 class TestUnitMetObsStationV1:
```
Comment (docNord) [inline tests/unit/test_unit_metobs.py:424]: Shouldn't these names follow suit?

File: tests/unit/test_unit_metobs.py [hunk 2]
```diff
@@ -603,8 +603,8 @@ class TestUnitMetObsDataV1:
             ([], "corrected-archive", "json"),
         ],
     )
-    @patch("smhi.metobs.MetobsLevelV1._get_and_parse_request")
-    @patch("smhi.metobs.MetobsLevelV1._get_url")
+    @patch("smhi.metobs.BaseLevel._get_and_parse_request")
+    @patch("smhi.metobs.BaseLevel._get_url")
     def test_unit_metobsdatav1_init(
```
Comment (docNord) [inline tests/unit/test_unit_metobs.py:608]: ...and these?

**Review by docNord — COMMENTED** [review]
All in all good changes which also makes the code easier to read. Made a few comments on changing the names of the test classes and functions to follow suit, that will probably help understanding in the future. :) Good job!

File: tests/unit/test_unit_metobs.py (hunk 1 above)
Comment (mgcth) [reply tests/unit/test_unit_metobs.py]: Fixed in [8fabb13](https://github.com/Ingenjorsarbete-For-Klimatet/ifk-smhi/pull/42/commits/8fabb135cb4a7a04d8922c2d9dd0b18d60ad0866).

File: tests/unit/test_unit_metobs.py (hunk 2 above)
Comment (mgcth) [reply tests/unit/test_unit_metobs.py]: Should be addressed.

**Review by docNord — APPROVED** [review]
Awesome 👌

**Pull request merged**